### ISSUE TITLE
Issue referring to label as tbl when no object

### ISSUE THREAD

**Issue by Schiano-NOAA** [nmfs-ost/asar#117]
The label for this chunk seems to throw an error code when rendering 

`WARNING (C:/Users/SAMANT~1.SCH/AppData/Local/Programs/Quarto/share/filters/main.lua:22058) FloatRefTarget with no content: tbl-indices-setup`

While it does not prevent the file from rendering, I think just removing the label will make more sense for this situation. Again, not sure why this is only a table thing.

https://github.com/nmfs-ost/asar/blob/807ae57aab6f35a1ff080ee783e8fc2cedcaedbe/R/create_tables_doc.R#L58

**Comment by sbreitbart-NOAA** [comment]
I'm having trouble reproducing this message and have rendered multiple reports without it. Does this error arise for every dataset you use? Or if not, can you describe when it's triggered?

**Comment by Schiano-NOAA** [comment]
@sbreitbart-NOAA Yeah! Let me see if it comes up again. I haven't rendered anything with the updated code since it wasn't updating for me last night, so I will let you know if this persists

**Comment by sbreitbart-NOAA** [comment]
@Schiano-NOAA this issue was resolved, right?

**Comment by Schiano-NOAA** [comment]
@sbreitbart-NOAA Yes, I believe so
